torch/utils/collect_env.py

```diff
@@ -567,6 +708,10 @@ def get_version_or_na(cfg, prefix):
 GPU models and configuration: {nvidia_gpu_models}
 Nvidia driver version: {nvidia_driver_version}
 cuDNN version: {cudnn_version}
+Is XPU available: {is_xpu_available}
+Intel GPU driver version: {intel_gpu_driver_version}
+Intel GPU models onboard: {intel_gpu_onboard}
+Intel GPU models detected: {intel_gpu_detected}
```

Comment:
It feels like it's probably a time to modify script to hide those behind `if_xyz_available`

Reply:
done. hide xpu info behind `is_xpu_available`.